enumerable.rb

```diff
@@ -0,0 +1,85 @@
+module Enumerable
+    def my_each
+        for item in self
+            yield(item)
+        end
+        self
+    end
```

Comment:
- [ ] returns an Enumerator if no block is given. 
You can do this `def my_each return to_enum unless block_given?`